Dockerfile

```diff
@@ -1,20 +1,42 @@
 FROM tomcat:9-jdk11-openjdk
+ARG USER_ID=3001
+ARG USER_NAME=vivo
+ARG USER_HOME_DIR=/home/$USER_NAME
 
-ARG SOLR_URL=http://localhost:8983/solr/vivocore
-ARG VIVO_DIR=/usr/local/vivo/home
-ARG TDB_FILE_MODE=direct
+ENV TOMCAT_CONTEXT_PATH=ROOT
+ENV VIVO_HOME_DIR=/usr/local/vivo/home
+ENV TDB_FILE_MODE=direct
+ENV ROOT_USER_ADDRESS=vivo_root@mydomain.edu
+ENV DEFAULT_NAMESPACE=http://vivo.mydomain.edu/individual/
 
-ENV SOLR_URL=${SOLR_URL}
-ENV JAVA_OPTS="${JAVA_OPTS} -Dtdb:fileMode=$TDB_FILE_MODE"
+ENV SOLR_URL=http://localhost:8983/solr/vivocore
+ENV SELF_ID_MATCHING_PROPERTY=http://vivo.mydomain.edu/ns
 
-RUN mkdir /usr/local/vivo
-RUN mkdir /usr/local/vivo/home
+ENV LOAD_SAMPLE_DATA=false
+ENV SAMPLE_DATA_REPO_URL=https://github.com/vivo-project/sample-data.git
+ENV SAMPLE_DATA_BRANCH=main
+ENV SAMPLE_DATA_DIRECTORY=openvivo
```

Comment:
I suppose this means openvivo data might be loaded in some reasonable time when load_sample_data flag is turned on (true).